File: python/prism_sdk/throughput_lease_fencing_integrity_workflow_fabric.py
"""Factory P32 throughput lease/fencing integrity workflow fabric."""
from .lease_fencing_integrity_support import *
FEATURE_ID="AFA-factory-P32-F15"; CONTRACT_VERSION="factory-throughput_lease_fencing_integrity_workflow_fabric/1.0"
def throughput_lease_fencing_integrity_workflow_fabric_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="workflow_fabric")
def qualify_throughput_lease_fencing_integrity_workflow_fabric(request):return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="workflow_fabric")

File: crates/factory/src/throughput_lease_fencing_integrity_workflow_fabric.rs
//! Factory P32 throughput lease/fencing integrity workflow fabric.
use super::lease_fencing_integrity_support::{
    manifest, qualify, LeaseFencingIntegrityCard7, LeaseFencingIntegrityError,
    LeaseFencingIntegrityRequest4,
};
pub const FEATURE_ID: &str = "AFA-factory-P32-F15";
pub const CONTRACT_VERSION: &str = "factory-throughput_lease_fencing_integrity_workflow_fabric/1.0";
pub fn throughput_lease_fencing_integrity_workflow_fabric_manifest() -> serde_json::Value {
    manifest(
        FEATURE_ID,
        CONTRACT_VERSION,
        "throughput",
        "workflow_fabric",
    )
}
pub fn qualify_throughput_lease_fencing_integrity_workflow_fabric(
    q: &LeaseFencingIntegrityRequest4,
) -> Result<LeaseFencingIntegrityCard7, LeaseFencingIntegrityError> {
    qualify(
        q,
        FEATURE_ID,
        CONTRACT_VERSION,
        "throughput",
        "workflow_fabric",
    )
}
